ext/pcntl/test-pcntl.php

```diff
@@ -35,6 +35,6 @@ function usr1_handle($signal){
 	print "Parent: Senging SIGUSR1 to Child\n";
 	posix_kill($pid,SIGUSR1);
 	sleep(2);
-	print "Parent: Sending SIGALRM to Child\n";
-	pcntl_waitpid($pid, &$status, $options);
+    print "Parent: Sending SIGALRM to Child\n";
+	pcntl_waitpid($pid, $status);
```

Comment:
- To fix syntax error, remove `&` from `$status`.
- Remove `$options` because this is an undefined variable and the `$options` is the optional argument.